### PR TITLE
Update file name and add check for failure. Timeout was checked already.

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

**Comment by lhellebr** [comment]
```
$ pytest tests/foreman/sys/test_dynflow.py
==================================================================== test session starts ====================================================================
platform linux -- Python 3.9.16, pytest-7.2.1, pluggy-1.0.0 -- /home/lhellebr/git/robottelo/venv/bin/python
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/lhellebr/git/robottelo, configfile: pytest.ini
plugins: services-2.2.1, reportportal-5.1.3, xdist-3.1.0, mock-3.10.0, ibutsu-2.2.4
collected 1 item                                                                                                                                            

tests/foreman/sys/test_dynflow.py::test_positive_setup_dynflow PASSED                                                                                 [100%]

=============================================================== 1 passed in 69.04s (0:01:09) ================================================================

```

**Review by Griffin-Sullivan — APPROVED** [review]
Looks good! Do we need to add an assert after https://github.com/SatelliteQE/robottelo/pull/10611/files#diff-fbad2e1a6d7b24a6f1bca2eb47d92c1f76fb8ae717fbc33e4e0f03e7828c1a9bR41?

**Comment by lhellebr** [comment]
That's just cleanup

File: tests/foreman/sys/test_dynflow.py [hunk 1]
```diff
@@ -29,13 +29,13 @@ def test_positive_setup_dynflow(target_sat):
...
         'sed -i s/5/6/ test.yml',
         "systemctl restart 'dynflow-sidekiq@test'",
         "while ! systemctl status 'dynflow-sidekiq@test' -l | "  # no comma here
         "grep -q ' of 6 busy' ; do sleep 0.5 ; done",
     ]
     # if thread count is not respected or the process is not running, this should timeout
-    # how is this a test? Nothing is asserted.
-    target_sat.execute(' && '.join(commands))
+    res = target_sat.execute(' && '.join(commands))
+    assert res.status == 0
```
Comment (Gauravtalreja1) [inline tests/foreman/sys/test_dynflow.py:40]:
```suggestion
    assert target_sat.execute(' && '.join(commands)).status == 0
```

**Review by Gauravtalreja1 — COMMENTED** [review]
Ack, with small suggestion

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**